### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by t0xicCode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-Installation
-------------
+## Overview
 
-To install, download the `taskcluster` binary for your platform from the latest
-release on [the releases page](https://github.com/taskcluster/taskcluster-cli/releases).
+TaskCluster CLI is a command-line client offering control and access to
+taskcluster from the comfort of your command-line. It provides utilities
+ranging from direct calls to the specific API endpoints to more complex and
+_useful_ tasks like listing cancelling scheduled runs.
```
Comment (t0xicCode) [inline README.md]: This might not be the best word. I might need a better word here.

Comment (yannlandry) [inline README.md]: Weird wording indeed. Maybe look into _advanced_, or _time-saving_. Requesting help from native English staff here!

Also, did you forget a word there?

> like listing _**and**_ cancelling scheduled runs

Comment (djmitche) [inline README.md]: "practical"?

Comment (yannlandry) [inline README.md]: ^ _This guy knows._

I like it.

**Pull request merged**